.circleci/scripts/verify-docs.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Verifies that docs have been generated and updated
+GREEN='\e[0;32m'
+RED='\033[0;31m'
+NC='\033[0m' # No Color
+
+md5Command=md5sum
+if [ $(uname -s) == "Darwin" ]; then
+    md5Command=(md5 -q)
+fi
+
+# Store MD5 of Committed Documentation
+MD5_ROOT_README_BEFORE=$(find README.md -type f -exec $md5Command {} \; | sort -k 2 | $md5Command)
```

Comment:
`-type f` - suppesed to check if README.md could be a directory? It does not seem to be needed here.